tests/infra/health_watcher.py

```diff
@@ -83,6 +85,19 @@ def get_network_health(network, get_primary_fn, client_node_timeout_s=3, verbose
         # about to lose its primaryship
         return HealthState.election
 
+    # The current primary is one-way partitioned from f backups, i.e. cannot
+    # receive acks from a majority of backups
+    # Note: This can be removed once the CheckQuorum extension is implemented
+    # as the partitioned primary node will automatically step down.
+    delayed_acks = [
```

Comment:
From a single operator's point-of-view, a network needs to be recovered if the primary doesn't receive any ACKs from a majority of backups for a long time, which would be the sign of one-way partitions between the primary and backups.